### PR TITLE
card styling changed

### PR DESCRIPTION
**Pull request by ganesh-kanchi**

### CODE REVIEW

**Review by Sen-442b — APPROVED** [review]
Looks clean to me

File: pages/cart-page/cart.css [hunk 1]
```diff
@@ -164,6 +147,7 @@ main{
 }
 
 .cart-transactions button {
+    margin: auto;
     margin-top: 8px;
```
Comment (prachi-sahani) [inline pages/cart-page/cart.css:151]: this can be combined (for eg: margin: 8px auto 0; - in case you don't need bottom margin)

**Review by prachi-sahani — COMMENTED** [review]
code looks good to merge 👍🏻

File: pages/wishlist-page/wishlist.html [hunk 2]
```diff
@@ -5,10 +5,8 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Wishlist JacketEX</title>
-    <link rel="stylesheet" href="https://lazylibrary.netlify.app/components.css">
-    <link rel="stylesheet" href="../../styles.css">
-    <link rel="stylesheet" href="../product-listing-page/products.css">
-    <link rel="stylesheet" href="wishlist.css">
+
```
Comment (krate27) [inline pages/wishlist-page/wishlist.html:8]: You can include all the css in one global file and link that in html

**Pull request merged**